p4-16/spec/P4-16-spec.mdk

```diff
@@ -3701,6 +3701,9 @@ list expressions.
 tuple<bit<32>, bool> x = { 10, false };
 ~ End P4Example
 
+The fields of a tuple are named `f0`, `f1`, etc,; they can be
```

Comment:
Should be "etc." not "etc,"

Reply:
Technically one dot is for the etc abbreviation, and the second is the sentence-ending dot.